applications/machine_learning/CMakeLists.txt

```diff
@@ -7,49 +7,18 @@
 cmake_minimum_required(VERSION 3.20.0)
 
 
-if (NOT CMAKE_BUILD_TYPE)
-  set(CMAKE_BUILD_TYPE ZDebug)
-endif()
-
 if (NOT EXISTS "${CMAKE_CURRENT_SOURCE_DIR}/configuration/${BOARD}")
   message(FATAL_ERROR
           "Board ${BOARD} is not supported.\n"
           "Please make sure board specific configuration files are added to "
           "${CMAKE_CURRENT_SOURCE_DIR}/configuration/${BOARD}")
 endif()
 
-if (NOT EXISTS "${CMAKE_CURRENT_SOURCE_DIR}/configuration/${BOARD}/app_${CMAKE_BUILD_TYPE}.conf")
-  message(FATAL_ERROR
-          "Configuration file for build type ${CMAKE_BUILD_TYPE} is missing.\n"
-          "Please add file ${CMAKE_CURRENT_SOURCE_DIR}/configuration/${BOARD}/app_${CMAKE_BUILD_TYPE}.conf")
-endif()
-
-if (EXISTS "${CMAKE_CURRENT_SOURCE_DIR}/configuration/${BOARD}/dts.overlay")
-  set(DTC_OVERLAY_FILE "${CMAKE_CURRENT_SOURCE_DIR}/configuration/${BOARD}/dts.overlay")
-endif()
-
-# Use dedicated configuration of MCUboot bootloader only if it's provided for given board and
-# build type. For example, application uses default configuration of the bootloader for Thingy:53.
-if (EXISTS ${CMAKE_CURRENT_SOURCE_DIR}/configuration/${BOARD}/mcuboot_${CMAKE_BUILD_TYPE}.conf)
-  set(mcuboot_CONF_FILE
-    ${CMAKE_CURRENT_SOURCE_DIR}/configuration/${BOARD}/mcuboot_${CMAKE_BUILD_TYPE}.conf
-    )
-endif()
-
-set(hci_rpmsg_CONF_FILE
-  ${CMAKE_CURRENT_SOURCE_DIR}/configuration/${BOARD}/hci_rpmsg_${CMAKE_BUILD_TYPE}.conf
-  )
-
-# Use static partition layout to ensure consistency between builds.
-# This is to ensure settings storage will be at the same location after the DFU.
-set(PM_STATIC_YML_FILE
-  ${CMAKE_CURRENT_SOURCE_DIR}/configuration/${BOARD}/pm_static_${CMAKE_BUILD_TYPE}.yml
-  )
+################################################################################
 
-# Define configuration files.
```

Comment:
There are some issues with naming convention for Thingy:53 files. Cannot add comments directly to renames so I list them here.

Reply:
Should we use name `prj_debug_nus.conf` or `prj_nus.conf`? nRF Desktop uses the second naming convention while ML app uses the first one currently.

Reply:
The PM static configuration file for `debug_rtt` configuration is missing (there is `pm_static` file that probably should be used for that purpose). For both secure and non-secure build variant.

Reply:
> Should we use name prj_debug_nus.conf or prj_nus.conf? nRF Desktop uses the second naming convention while ML app uses the first one currently.

The effect of scripting. That should be `nus`. I don't want to repeat `debug` everywhere.

Reply:
> The PM static configuration file for debug_rtt configuration is missing (there is pm_static file that probably should be used for that purpose). For both secure and non-secure build variant.

Was it missing from the start? I have not removed any files - script only renamed them

Reply:
btw - when there is no pm_static specific file the default pm_static will be used. This is why I left it as is.

Reply:
I think renaming with script handled something improperly.

Reply:
We could consider defining only a single pm_static then (consistently for both variants of Thingy). Both versions are acceptable for me, but I would prefer to keep in constistent.

Reply:
fixed

Reply:
The static PM definition is still not fixed for Thingy:53 (both secure and non-secure variant)

Reply:
> The pm_static_rtt.yml, seems to be named pm_static

missed that :/

Reply:
should be fixed now